exchanges/okx/okx_test.go

```diff
@@ -0,0 +1,3129 @@
+	t.Parallel()
+	if err := ok.TickersSubscription("subscribe", asset.Spot, currency.NewPair(currency.BTC, currency.USDT)); err != nil {
+		t.Errorf("%s TickersSubscription() error: %v", ok.Name, err)
+	}
+	if err := ok.TickersSubscription("unsubscribe", asset.Spot, currency.NewPair(currency.BTC, currency.USDT)); err != nil {
+		t.Errorf("%s TickersSubscription() error: %v", ok.Name, err)
+	}
+}
+func TestOpenInterestSubscription(t *testing.T) {
+	t.Parallel()
+	if err := ok.OpenInterestSubscription("subscribe", asset.PerpetualSwap, currency.NewPair(currency.BTC, currency.NewCode("USD-SWAP"))); err != nil {
+		t.Errorf("%s OpenInterestSubscription() error: %v", ok.Name, err)
+	}
+}
+func TestCandlesticksSubscription(t *testing.T) {
+	t.Parallel()
+	enabled, err := ok.GetEnabledPairs(asset.PerpetualSwap)
+	if err != nil {
+		t.Error("couldn't find enabled tradable pairs")
+	}
+	if len(enabled) == 0 {
+		t.SkipNow()
+	}
+	if err := ok.CandlesticksSubscription("subscribe", okxChannelCandle1m, asset.Futures, enabled[0]); err != nil {
+		t.Errorf("%s CandlesticksSubscription() error: %v", ok.Name, err)
+	}
+}
+
+func TestTradesSubscription(t *testing.T) {
+	t.Parallel()
+	if err := ok.TradesSubscription("subscribe", asset.Spot, currency.NewPair(currency.BTC, currency.USDT)); err != nil {
+		t.Errorf("%s TradesSubscription() error: %v", ok.Name, err)
+	}
+}
+
+func TestEstimatedDeliveryExercisePriceSubscription(t *testing.T) {
+	t.Parallel()
+	futuresPairs, err := ok.FetchTradablePairs(context.Background(), asset.Futures)
+	if err != nil {
+		t.Errorf("%s error while fetching tradable pairs for instrument type %v: %v", ok.Name, asset.Futures, err)
+	}
+	if len(futuresPairs) == 0 {
+		t.SkipNow()
+	}
+	currencyPair, err := currency.NewPairFromString(futuresPairs[0])
+	if err != nil {
+		t.Error(err)
+	}
+	if err := ok.EstimatedDeliveryExercisePriceSubscription("subscribe", asset.Futures, currencyPair); err != nil {
+		t.Errorf("%s EstimatedDeliveryExercisePriceSubscription() error: %v", ok.Name, err)
+	}
+}
+
+func TestMarkPriceSubscription(t *testing.T) {
+	t.Parallel()
+	futuresPairs, err := ok.FetchTradablePairs(context.Background(), asset.Futures)
+	if err != nil {
+		t.Errorf("%s error while fetching tradable pairs for instrument type %v: %v", ok.Name, asset.Futures, err)
+	}
+	if len(futuresPairs) == 0 {
+		t.SkipNow()
+	}
+	currencyPair, err := currency.NewPairFromString(futuresPairs[0])
+	if err != nil {
+		t.Error(err)
+	}
+	if err := ok.MarkPriceSubscription("subscribe", asset.Futures, currencyPair); err != nil {
+		t.Errorf("%s MarkPriceSubscription() error: %v", ok.Name, err)
+	}
+}
+
+func TestMarkPriceCandlesticksSubscription(t *testing.T) {
+	t.Parallel()
+	enabled, err := ok.GetEnabledPairs(asset.Spot)
+	if err != nil {
+		t.Error("couldn't find enabled tradable pairs")
+	}
+	if len(enabled) == 0 {
+		t.SkipNow()
+	}
+	if err := ok.MarkPriceSubscription("subscribe", asset.Futures, enabled[0]); err != nil {
```

Comment:
```suggestion
	if err := ok.MarkPriceCandlesticksSubscription("subscribe", asset.Futures, enabled[0]); err != nil {
```
You called the wrong function